src/itest/kotlin/nl/lifely/zac/itest/ZaakRestServiceCompleteTest.kt

```diff
@@ -62,6 +56,14 @@ class ZaakRestServiceCompleteTest : BehaviorSpec({
                 intakeId = getString("id").toInt()
             }
         }
+        // Wait before setting the status of a zaak (implicitly)
+        // because OpenZaak does not allow setting multiple statuses for one zaak
+        // within the same timeframe of one second.
+        // If we do not wait in these cases we get a 400 response from OpenZaak with:
+        // "rest_framework.exceptions.ValidationError: {'non_field_errors':
+        // [ErrorDetail(string='De velden zaak, datum_status_gezet moeten een unieke set zijn.', code='unique')]}"
+        //
+        // Related OpenZaak issue: https://github.com/open-zaak/open-zaak/issues/1639
         sleep(1)
```

Comment:
Wouldn't `eventually` help here? Eventually we'll get non-400 response when it's ready to proceed

Reply:
No, afraid not. We tried that. Can't remember why but it did not work.